src/main/java/com/minecolonies/coremod/sounds/KnightSounds.java

```diff
@@ -117,15 +118,15 @@ public void playSound(final World worldIn, final BlockPos position, final boolea
     @Override
     public String getWorkerString()
     {
-        return "GuardTower";
+        return "knight";
     }
 
     /**
      * Play interaction sound.
      *
-     * @param worldIn    world to play it in.
-     * @param position   position to play it at.
-     * @param isFemale   the gender.
+     * @param worldIn  world to play it in.
+     * @param position position to play it at.
+     * @param isFemale the gender.
      */
     public void playInteractionSound(final World worldIn, final BlockPos position, final boolean isFemale)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Provide multiple methods instead of using "isFemale" to determine which action to take. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2301)